### DISCUSSION TITLE
Stacktorm Workflow accessing variable from sensor

### DISCUSSION THREAD

**Discussion by ManuGeodis** [StackStorm/st2#5904] [Workflows]
I'm having a sensor which sends payload values to rules and its working fine. I need to use that payload value in a workflow. Can you give me an example how to use the payload value from sensor in workflow?

@armab

**Comment by arm4b** [comment]
Converted to Discussion for community to answer.

**Comment by martezr** [comment]
@ManuGeodis The rules documentation (https://docs.stackstorm.com/rules.html) includes several examples of passing a payload value from the sensor to the associated workflow.

**Reply by ManuGeodis** [reply]
@martezr The payload value is passed to the rule in that document. In /actions/workflow/, I must send the payload value to a workflow.

**Reply by ManuGeodis** [reply]
@armab @martezr  How to pass a variable from rule to workflows in orquesta?

**Reply by ManuGeodis** [reply]
I have passed the variable with the parameter 'parameter' in the rule and workflow meta.
